src/all/meituatop/build.gradle

```diff
@@ -0,0 +1,8 @@
+ext {
+    extName = 'Meitua.top'
+    extClass = '.MeituaTop'
+    extVersionCode = 7

```

Comment:
You don't need to bump the version if you're not changing anything. Although that's hard to tell through GitHub's UI.